server/adaptors/integrations/__data__/repository/k8s/info/README.md

```diff
@@ -0,0 +1,42 @@
+# Kubernetes Integration
```

Comment:
I'm a little concerned about a structure with 3 READMEs in subdirectories and no main one, is there anything we can do to reorganize?

Reply:
we should probably have a parent one directing for all subjects that are also accompanied with a readme file - TDB for the next release